manifests/cloner/script.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if [ "$1" == "source" ] ; then
+  mkfifo /tmp/clone/socket/pipe; cat /tmp/clone/image/disk.img > /tmp/clone/socket/pipe
+elif [ "$1" == "target" ] ; then
```

Comment:
Add echo "Starting clone target"

Reply:
Done

Reply:
yes, and add comment describing $1 and $2 args.